src/style.css

```diff
@@ -0,0 +1,37 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  width: 96;
```

Comment:
This value is incorrect, you can't just set number, need to specify unit (px in our case)